Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -395,6 +395,14 @@ HRESULT _DWriteCreateFontFaceWithName(CFStringRef name, IDWriteFontFace** outFon
     // Eg: Bold, Condensed, Light, Italic
     _DWriteFontProperties properties = _DWriteGetFontPropertiesFromName(name);
 
+    // TODO: #1250: Need to be able to load fonts from the app's bundle
+    // For now return a default font to avoid crashes in case of missing fonts
+    // When #1250 is completed, remove this
+    if (!properties.familyName) {
+        name = CFSTR("Segoe UI");
```

Comment:
no.
- we're reverting this soon anyway, no reason to make this change bigger and messier to revert.
- this isn't intended to be our go-to default font functionality. we have other default font constants elsewhere. this just has to be any default font. (@rajsesh-msft and I were throwing around the idea of using something ugly so it'd stick out)

---
In reply to: [85445534](https://github.com/Microsoft/WinObjC/pull/1253#discussion_r85445534) [](ancestors = 85445534)